source/templates.tex

```diff
@@ -694,18 +694,13 @@
   id-expression
 \end{bnf}
 
-\begin{note}
-The name lookup rules\iref{basic.lookup} are used to associate the use of
-a name with a template declaration;
-that is, to identify a name as a
-\grammarterm{template-name}.
-\end{note}
-
 \pnum
-For a
-\grammarterm{template-name}
-to be explicitly qualified by the template arguments,
-the name must be considered to refer to a template.
+An identifier is a \grammarterm{template-name}
```

Comment:
```suggestion
An \grammarterm{identifier} is a \grammarterm{template-name}
```

Reply:
I initially grammarterm'ed these identifiers, but we never talk about a grammarterm being another grammarterm; we speak of it as having the form of another grammarterm.

Suggestions applied nonetheless.